### ISSUE TITLE
Make sure tab panels are included in page sequence

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#840]
**Page 55 in DAC report**

**Issue from DAC report:**
Tab-panel content does not take focus upon activation. Tabs have been implemented as same page links, but these do not work and upon activation, focus is taken to the other tabs before the tab-panel content.

Tab-panels are not focusable and screen reader users are not able to navigate from the tabs using cursor keys, so in order to read the content within the tab it is necessary to tab to the next focusable element on the page, and then use the cursor keys to navigate back up the page.

**Solution from DAC report:**
Include the tabpanel in the page sequence so that it can be focused by screen reader users. Ensure that focus is taken to the tabpanel upon activation.

**Example from DAC report:**

```
<div class="tab-content">
    <div role="tabpanel" class="tab-pane active" id="national"
tabindex="0">…</div>
</div>
```

Includes data tabpanel, metadata tabpanel and reporting status tab-panel

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning added this by just realised it's on our list to discuss with DAC for clarification so might be worth holding off on this one for now

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning apologies this is another one I should have raised on call. Discussed this with DAC this morning, and they said tabindex=0 should only be when a tab is active. I think in the PR, tabindex=0 was added to all. Sorry for not clarifying before approving. It appeared that it was working as report described it should be

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin Aha, yes we need to revisit this. And in fact I was confused before. I see now that, before merging that PR, it actually was possible to use the keyboard to get into the tab content - you just have to get past the other tabs first. Now since we merged that PR, once you activate a tab, the keyboard navigation drops you down past all the tabs. Tough to describe, especially with the dual-meaning of "tab"...

Ideally, the keyboard navigation would go on to the next tab after you have tabbed through all the links in the active tab's content.

I'll open a PR to revert that last PR for now.
